qgreenland/util/raster.py

```diff
@@ -101,7 +102,15 @@ def _gdalwarp_cut_hack(out_path, inp_path, *, layer_cfg, warp_kwargs):
 
     # Step 1 needs to subset for this to work (outputBounds == `-te`).
     step1_kwargs = {k: v for k, v in warp_kwargs.items() if k not in step2_keys}
-    step1_kwargs['outputBounds'] = layer_cfg['boundary']['bbox']
+
+    # HACK ANOTHER TERRIBLE HACK: reprojection and clipping should really be two
+    # separate steps!!!
```

Comment:
>reprojection and clipping should really be two separate steps!!!

Excellent point. Is there any reason we couldn't have two Warp steps stacked on top of each other? I can't think of one.